src/libs/actions/Policy/Policy.ts

```diff
@@ -2251,30 +2254,14 @@ function buildPolicyData(options: BuildPolicyDataOptions = {}) {
     return {successData, optimisticData, failureData, params};
 }
 
-function createWorkspace(
```

Comment:
i changed the code here from individual param to an object because of,
```
Error:   2260:1   error    Function 'createWorkspace' has too many parameters (11). Maximum allowed is 10                               @typescript-eslint/max-params
```